lib/perl5db.pl

```diff
@@ -2055,22 +2048,13 @@ sub _DB__handle_c_command {
                 # Mark that there's a breakpoint in this file.
                 $had_breakpoints{$filename} |= 1;
 
-                # Scan forward to the first executable line
-                # after the 'sub whatever' line.
-                $max = $#dbline;
-                my $_line_num = $i;
-                while ($dbline[$_line_num] == 0 && $_line_num< $max)
-                {
-                    $_line_num++;
-                }
-                $i = $_line_num;
+                $i = $line;
             } ## end if ($i)
```

Comment:
Just noticed this: The comment should be either removed or changed to `end if ($line)`.